src/trilateration/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+
+#thruster nodes
+add_ros_node(hydrophones trilateration.cpp)
```

Comment:
This seems a bit overkill for just a single node, moving to the `sensors` subdirectory would condense things.